src/views/DocumentCompare.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+	<div class="bg-gray-300">
+		<ResultHeader v-if="documents" title="Podobnosť dokumentov:">
+			<p slot="description" class="hidden md:flex text-xl md:invisible">
+				{{ documents.textA.name }}
+				<span class="font-medium px-2"> a </span>
+				{{ documents.textB.name }}
+			</p>
+		</ResultHeader>
+		<div v-if="documents" class="px-5 pt-5 md:px-10 md:pt-0">
+			<a
+				class="flex items-center p-2 mb-0 md:mb-2 cursor-pointer text-lg md:text-2xl text-gray-600 hover:text-gray-700"
+				@click="
+					$router.push({
+						name: `document`,
+						params: {result:resultId, document: documentId}
+					})
+				"
+			>
+				<i class="fas fa-chevron-left text-sm md:text-lg mr-2" />
+				Späť
+			</a>
+		</div>
+		<div v-if="documents" class="md:flex py-4 md:py-4">
+			<div
+				class="md:flex1 rounded-md md:rounded-lg mx-4 md:mt-0 md:mr-2 md:ml-4 md:w-1/2 lg:ml-12"
+			>
+				<div
+					class="flex items-center justify-between px-2 py-1 md:p-2 md:h-12 bg-primary-500 text-white text-lg rounded-t-md md:rounded-t-lg"
+				>
+					<p class="mx-auto">
+						{{ documents.textA.name }}
+					</p>
+				</div>
+				<div
+					v-html="highlightedTexts.A"
+					class="px-4 py-1 pt-1 md:pt-2 md:p-4 bg-white text-justify rounded-b-md md:rounded-b-lg shadow-md"
+				/>
+			</div>
+			<div
+				class="md:flex1 rounded-md md:rounded-md mt-4 mx-4 md:mt-0 d:mr-4 md:ml-2 md:w-1/2 lg:mr-12"
+			>
+				<div
+					class="flex items-center justify--between px-2 py-1 md:p-2 md:h-12 bg-primary-500 text-white text-lg rounded-t-md md:rounded-t-lg"
+				>
+					<p class="mx-auto">
+						{{ documents.textB.name }}
+					</p>
+				</div>
+				<div
+					v-html="highlightedTexts.B"
+					class="px-4 py-1 pt-1 md:pt-2 md:p-4 bg-white text-justify rounded-b-md md:rounded-b-lg shadow-md"
+				/>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import ResultHeader from '../components/Result/ResultHeader';
+import { escape, cloneDeep } from 'lodash';
+import retry from '@/functions/retry.function';
+import { colorForIndex } from '@/utilities/color.utility';
+
+export default {
+	components: {
+		ResultHeader,
+	},
+	data: function () {
+		return {
+			id: undefined,
+			documentId: undefined,
+			resultId: undefined,
+			highlightedTexts: undefined,
+			documents: undefined,
+		};
+	},
+	mounted: function () {
+		// initial spinner
+		this.$store.dispatch('setLoading', true);
+
+		this.resultId = this.$route.params.result;
+		this.id = this.$route.params.compare;
```

Comment:
NIT: `documentBId` (You're using the terms A and B everywhere in the code, why not in id's